src/libs/OptionsListUtils.js

```diff
@@ -204,20 +204,41 @@ function hasReportDraftComment(report) {
         && lodashGet(reportsWithDraft, `${ONYXKEYS.COLLECTION.REPORTS_WITH_DRAFT}${report.reportID}`, false);
 }
 
+/**
+ * @param {Object} report
+ * @returns {Array<String>}
+ */
+function getParticipants(report) {
+    const participants = [...lodashGet(report, ['participants'], [])];
+    if (ReportUtils.isPolicyExpenseChat(report)) {
+        if (!participants.includes(report.ownerEmail)) {
+            participants.push(report.ownerEmail);
+        }
+
+        // Include admin as participant
+        if (!report.isOwnPolicyExpenseChat) {
+            participants.push(currentUserLogin);
+        }
+    }
+    return participants;
+}
```

Comment:
I created this function to include the admin and/or the workspace member as participants in order to include the name on the second line of the chat header (i.e. "User1: test message").

Reply:
I think the admin should be a participant. Any ideas on why they not showing up in the participants array?

Reply:
Report owner makes sense as they do not have a `sharedReports` record for their own policy expense chat.

Reply:
Ohh that's right, I realized it was excluded here: https://github.com/Expensify/App/blob/36c6f8a2ccb4a846daff664036571c2cdf1fc164/src/libs/actions/Report.js#L126

I fixed it and removed this condition in `getParticipants`